llama_stack/distribution/build_venv.sh

```diff
@@ -35,12 +35,10 @@ set -euo pipefail
 
 build_name="$1"
 env_name="llamastack-$build_name"
-build_file_path="$2"
```

Comment:
this is not needed? should we not accept this parameter then?

Reply:
this param was never used so I removed it.